Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
+  run:
+    cmds:
+      - go run main.go
+  dev_server:
+    cmds:
```

Comment:
Each `go run` command references external packages. Ensure that the versions specified (like '@v1.61.1') are kept up to date to avoid using outdated code that may contain vulnerabilities or performance issues.